docs/quick-ingestion-guides/tableau/setup.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Setup
+---
+# Tableau Ingestion Guide: Setup & Prerequisites
+
+In order to configure ingestion from Tableau, you'll first have to enable Tableau Metadata API and you should have a user with Site Administrator Explorer permissions.
```

Comment:
In order to configure ingestion from Tableau, you'll first need to enable the Tableau Metadata API and obtain a user with Site Administrator Explorer permissions.

Reply:
Question - how do users do this? Can we expect that they know how to create this user?

Reply:
Added steps for how to assign `Site Administrator Explorer` permission to new user